clients/p4rt-ctl/p4rt-ctl.in

```diff
@@ -81,6 +81,12 @@ def usage():
     print(USAGE)
     sys.exit(0)
 
+def valid_ip(address):
+    try:
+        socket.inet_aton(address)
```

Comment:
This only works for IPv4 addresses. Need to fix this to handle both IPv4 and IPv6 addresses

Reply:
There is a `matchesIPv6(ipv6_str)` method in this file which can be used

Reply:
Have we verified that the Stratum gRPC interface supports both IPv4 and IPv6 addresses?

Reply:
Yes, IPv6 feature support was completed 0.9.0

Reply:
It's more about whether IPV6 addresses are supported for P4runtime server addresses. IPV6 support we added was for interfaces(eg. MEV netdevs or vhost-user ports in case of DPDK)

gRPC supports both IPv4 and IPV6 addresses for listening ports. Not sure, about Stratum. Will do some testing and get back

Reply:
> It's about whether IPV6 addresses are supported for P4runtime server addresses.

Thank you. That's the question I was asking.

Reply:
ok, was under the impression that IPv6 support also existed for infrap4d/stratum gRPC interface.

Reply:
Just confirmed with Richa as well that it wasn't added as part of IPV6 support addition. We should add this task to the backlog since grpc supports both